group_vars/hyperchicken_cluster/vars.yml

```diff
@@ -59,8 +59,8 @@ ldap_domains:
     group_member: member
     create_ldap: true
     ldap_db_index: 3  # Indices up to 2 are already used by default for the "config", "monitor" and "example" databases.
-cloud_image: CentOS7-Cloud
-cloud_user: centos
+cloud_image: RockyLinux-9.3_xfs
+local_volume_size_root: 22
```

Comment:
I am confused a bit of this root volumes. In some cases we have image that comes with predefined root volume, and that image is stored on server's disk (which can be in reality a network storage, but that's besides the point).
And in some cases like in Embassy it seems that you pick an image, which is probably just an OS compressed (like iso image), but then you must provide a root volume extra size to write that OS image into.

Do I get this correct? Or am I mixing things?

Reply:
Some (but not all) of the flavors we use come with a default size for disk. But `local_volume_size_root` can overrule this. Obviously `local_volume_size_root` must be large enough to contain the OS image used. It may be larger than the image though. We also have `local_volume_size_extra`, which is for the first "extra" disk; usually /dev/vdb for VMs or /dev/sdb for bare metals.
The current deployments of Rocky 9 take ~ 9 GB. I reduced `local_volume_size_root` from 33 to 22 for Hyperchicken, because we ran out of quota and given that it is only a test cluster I expect 22 should provide a large enough buffer.